src/scraper.py

```diff
@@ -631,70 +789,78 @@ def login(email, password):
+            )
         except Exception:
-            print("Kindly replace the Chrome Web Driver with the latest one from "
-                  "http://chromedriver.chromium.org/downloads "
-                  "and also make sure you have the latest Chrome Browser version."
-                  "\nYour OS: {}".format(platform_)
-                  )
-            exit()
+            print(
+                "Kindly replace the Chrome Web Driver with the latest one from "
+                "http://chromedriver.chromium.org/downloads "
+                "and also make sure you have the latest Chrome Browser version."
+                "\nYour OS: {}".format(platform_)
+            )
+            exit(1)
 
         fb_path = facebook_https_prefix + "facebook.com"
         driver.get(fb_path)
         driver.maximize_window()
 
         # filling the form
-        driver.find_element_by_name('email').send_keys(email)
-        driver.find_element_by_name('pass').send_keys(password)
+        driver.find_element_by_name("email").send_keys(email)
+        driver.find_element_by_name("pass").send_keys(password)
 
         # clicking on login button
-        driver.find_element_by_id('loginbutton').click()
+        driver.find_element_by_id("loginbutton").click()
 
         # if your account uses multi factor authentication
-        mfa_code_input = safe_find_element_by_id(driver, 'approvals_code')
+        mfa_code_input = safe_find_element_by_id(driver, "approvals_code")
 
         if mfa_code_input is None:
             return
 
         mfa_code_input.send_keys(input("Enter MFA code: "))
-        driver.find_element_by_id('checkpointSubmitButton').click()
+        driver.find_element_by_id("checkpointSubmitButton").click()
 
         # there are so many screens asking you to verify things. Just skip them all
-        while safe_find_element_by_id(driver, 'checkpointSubmitButton') is not None:
-            dont_save_browser_radio = safe_find_element_by_id(driver, 'u_0_3')
+        while safe_find_element_by_id(driver, "checkpointSubmitButton") is not None:
+            dont_save_browser_radio = safe_find_element_by_id(driver, "u_0_3")
             if dont_save_browser_radio is not None:
                 dont_save_browser_radio.click()
 
-            driver.find_element_by_id('checkpointSubmitButton').click()
+            driver.find_element_by_id("checkpointSubmitButton").click()
 
     except Exception:
         print("There's some error in log in.")
         print(sys.exc_info()[0])
-        exit()
+        exit(1)
 
 
 # -----------------------------------------------------------------------------
 # -----------------------------------------------------------------------------
 
+
 def main():
-    with open('credentials.txt') as f:
-        email = f.readline().split('"')[1]
-        password = f.readline().split('"')[1]
+    with open("credentials.yaml", "r") as ymlfile:
+        cfg = yaml.load(stream=ymlfile, Loader=yaml.BaseLoader)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load().](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4586331)